patch/kernel/sunxi-dev/packaging-4.x-dev.patch

```diff
@@ -1,16 +1,15 @@
 diff --git a/scripts/package/builddeb b/scripts/package/builddeb
-index 0bc8747..9dab810 100755
+index b4f0f2b..062ccf5 100755
 --- a/scripts/package/builddeb
 +++ b/scripts/package/builddeb
-@@ -29,6 +29,28 @@ create_package() {
+@@ -29,6 +29,27 @@ create_package() {
  	# in case we are in a restrictive umask environment like 0077
  	chmod -R a+rX "$pdir"
  
 +	# Create preinstall and post install script to remove dtb
 +	if [[ "$1" == *dtb* ]]; then
 +		echo "if [ -d /boot/dtb-$version ]; then mv /boot/dtb-$version /boot/dtb-$version.old; fi" >> $pdir/DEBIAN/preinst
 +		echo "if [ -d /boot/dtb.old ]; then rm -rf /boot/dtb.old; fi" >> $pdir/DEBIAN/preinst
-+		echo "if [ -d /dtb ]; then mv /dtb /dtb.old; fi" >> $pdir/DEBIAN/preinst
```

Comment:
Why removing this?

Reply:
Because it's not present in **patch/kernel/sunxi-next/packaging-4.x-next.patch** either. As is said this is almost copy/paste from sunxi-next branch.